packages/deprecation-crawler/src/lib/tasks/ensure-tsconfig-path.ts

```diff
@@ -0,0 +1,57 @@
+import { CrawlConfig } from '../models';
+import { prompt } from 'enquirer';
+import { normalize } from 'path';
+import { fileExists } from '@nrwl/workspace/src/utils/fileutils';
+import { updateRepoConfig } from '../utils';
+import { glob } from 'glob';
+import { CRAWLER_MODES } from '../constants';
+
+export async function ensureTsConfigPath(
+  config: CrawlConfig
+): Promise<CrawlConfig> {
+  // If a tsconfig files is already set proceed
+  if (fileExists(config.tsConfigPath)) {
+    console.log(`Crawling with tsconfig: ${config.tsConfigPath}`);
+    return config;
+  }
+
+  if (process.env.__CRAWLER_MODE__ === CRAWLER_MODES.SANDBOX) {
+    throw new Error(`${config.tsConfigPath} does not exist.`);
+  }
+
+  // Ensure any tsconfig files present to select
+  const tsConfigFiles: string[] = findTsConfigFiles();
+  if (tsConfigFiles.length === 0) {
+    throw Error(
+      'No tsconfig.json files present in the current folder structure.'
```

Comment:
An improvement to the DX we can make, is to provide more details here on how to create a tsconfig file, and why it's important.